### ISSUE TITLE
Crash after update

### ISSUE THREAD

**Issue by zethra** [kiwix/kiwix-android#3136]
**Describe the bug**

I updated to version 3.4.5 and the app now crashes on startup.

**Expected behavior**

Not crashing

**Steps to reproduce the behavior:**
1. Update the app. Not sure what version I was running before.

**Screenshots**

**Environment**
- Version of Kiwix Android : 3.4.5
- Device : Pixel 5
- OS version : Android 13 (no google services)

**Logs**

```
FATAL EXCEPTION: main
Process: org.kiwix.kiwixmobile, PID: 15947
java.lang.RuntimeException: Unable to create application org.kiwix.kiwixmobile.KiwixApp: io.objectbox.exception.DbSchemaException: DB's last entity ID 10 is higher than 8 from model
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6790)
	at android.app.ActivityThread.-$$Nest$mhandleBindApplication(Unknown Source:0)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:2134)
	at android.os.Handler.dispatchMessage(Handler.java:106)
	at android.os.Looper.loopOnce(Looper.java:201)
	at android.os.Looper.loop(Looper.java:288)
	at android.app.ActivityThread.main(ActivityThread.java:7898)
	at java.lang.reflect.Method.invoke(Native Method)
	at com.android.internal.os.RuntimeInit$MethodAndArgsCaller.run(RuntimeInit.java:548)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:936)
Caused by: io.objectbox.exception.DbSchemaException: DB's last entity ID 10 is higher than 8 from model
	at io.objectbox.BoxStore.nativeCreateWithFlatOptions(Native Method)
	at io.objectbox.BoxStore.<init>(BoxStore.java:20)
	at org.kiwix.kiwixmobile.core.di.modules.DatabaseModule_ProvidesBoxStoreFactory.get(DatabaseModule_ProvidesBoxStoreFactory.java:190)
	at dagger.internal.DoubleCheck.get(DoubleCheck.java:6)
	at org.kiwix.kiwixmobile.core.di.modules.DatabaseModule_ProvidesFetchDownloadDaoFactory.get(DatabaseModule_ProvidesFetchDownloadDaoFactory.java:1)
	at dagger.internal.DoubleCheck.get(DoubleCheck.java:6)
	at org.kiwix.kiwixmobile.core.downloader.fetch.FetchDownloadMonitor_Factory.get(FetchDownloadMonitor_Factory.java:1)
	at org.kiwix.kiwixmobile.core.di.modules.ApplicationModule_ProvideDownloadMonitor$core_releaseFactory.get(ApplicationModule_ProvideDownloadMonitor$core_releaseFactory.java:1)
	at dagger.internal.DoubleCheck.get(DoubleCheck.java:6)
	at org.kiwix.kiwixmobile.core.di.components.DaggerCoreComponent.inject(DaggerCoreComponent.java:5)
	at org.kiwix.kiwixmobile.core.CoreApp.onCreate(CoreApp.kt:12)
	at android.app.Instrumentation.callApplicationOnCreate(Instrumentation.java:1270)
	at android.app.ActivityThread.handleBindApplication(ActivityThread.java:6785)
	... 9 more
```

**Comment by zethra** [comment]
Unfortunately I didn't check before I updated. Probably a recent one, but I don't enable auto updates so it could have been a bit old.

**Comment by gouri-panda** [comment]
@zethra Ok. We'll investigate from here. Thanks for the feedback!

**Comment by kelson42** [comment]
@gouri-panda What could we do here, sounds pretty difficult, in particular @zethra does not use the latest version :(

**Comment by gouri-panda** [comment]
@kelson42 This happens when we rename an entity class or just a property. Without further information, ObjectBox will remove the old entity/property and add a new one with the new name.
There are basically two ways to resolve this:
* Resolve conflicts in or reconstruct the default.json meta model file to match the model of an existing database file.
* Keep or create a fresh default.json model file and delete the database file. However, this will lose all existing data.

**Comment by gouri-panda** [comment]
I'll place a fix there.

**Comment by kelson42** [comment]
@MohitMaliFtechiz Does that still make sense considering that objectbox db won’t be used anymore in 3.7.0 (except for migration to Room DB)? If « yes » why?

**Comment by MohitMaliFtechiz** [comment]
> @MohitMaliFtechiz Does that still make sense considering that objectbox db won’t be used anymore in 3.7.0 (except for migration to Room DB)? If « yes » why?

It only required for migration.

**Comment by kelson42** [comment]
@MohitMaliFtechiz If this is required for migration, then we should do it.

**Comment by kelson42** [comment]
Fixed by #3228
